workspaces/services/guacamole-azure-dsvm/terraform/deploy.sh

```diff
@@ -0,0 +1,8 @@
+export TF_LOG=""
+terraform init -input=false -backend=true -reconfigure \
+    -backend-config="resource_group_name=$TF_VAR_mgmt_resource_group_name" \
+    -backend-config="storage_account_name=$TF_VAR_mgmt_storage_account_name" \
+    -backend-config="container_name=$TF_VAR_terraform_state_container_name" \
+    -backend-config="key=${TF_VAR_tre_id}${TF_VAR_workspace_id}guacamolewindsvm"
```

Comment:
This should probably have parent service ID in it? Only used for running locally/testing at present, so don't think essential.